python/ql/src/Security/CWE-327/InsecureDefaultProtocol.qhelp

```diff
@@ -3,21 +3,22 @@
 "qhelp.dtd">
 <qhelp>
      <overview>
-          <p> The <code>ssl</code> library defaults to an insecure version of
-          SSL/TLS when no specific protocol version is specified. This may leave
-          the connection vulnerable to attack.
+          <p>
+          The <code>ssl</code> library defaults
```

Comment:
Since this query is only alerting on this one function, we can just mention it explicitly. 

```suggestion
          The <code>ssl.wrap_socket</code> function defaults
```